VogtObserver3/Test.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtObserver3
+{
+    class Test
+    {
+        public static void Run()
+        {
+            Store store = new Store();
+            Store store2 = new Store();
+
+            Customer cust = new Customer("Julie", "Bed Bath & Beyond");
+            Customer cust2 = new Customer("Doug", "Hot Topic");
+
+            store.AddObserver(cust);
+            store.AddObserver(cust2);
+            store2.AddObserver(cust2);
+
+
+            for (int i = 0; i < 20; i++)
+            {
+                store.GetDiscount();
+                store2.GetDiscount();
+            }
+
+
+            store.RemoveObserver(cust2);
```

Comment:
Why remove an observer and then do nothing? Seems like you'd want to loop through a few more times to make sure you truly removed the observer and they're no longer being notified? (For your own sake when debugging, at least)